### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by miyaliu666**
I didn't translate the commented lines.
And will translate the footer after it's updated on English version.

### CODE REVIEW

File: default.hbs [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
         {{!-- All the main content gets inserted here, index.hbs, post.hbs, etc --}}
         <div class="under-header-content jumbotron">
-            <h1 class="site-title">Welcome to Developer News.</h1>
+            <h1 class="site-title">文章</h1>
             <h2 class="site-description">
                 This is a free, open source, no-ads place to cross-post your blog articles. Read about it
                 <a href="https://www.freecodecamp.org/forum/t/279929">here</a>.
```
Comment (raisedadead) [inline default.hbs]: We would want to translate and point to the Chinese forum link here instead? But that can be taken up separate of this pull-request.

File: error-404.hbs [hunk 2]
```diff
@@ -34,7 +34,7 @@ It's a good idea to keep this template as minimal as possible in terms of both f
 
         {{> "site-nav"}}
         <div class="under-header-content jumbotron">
-            <h1 class="site-title">Welcome to Developer News.</h1>
+            <h1 class="site-title">欢迎来到 freeCodeCamp 社区</h1>
             <h2 class="site-description">
                 This is a free, open source, no-ads place to cross-post your blog articles. Read about it
                 <a href="https://www.freecodecamp.org/forum/t/279929">here</a>.
```
Comment (raisedadead) [inline error-404.hbs]: Same as https://github.com/freeCodeCamp/news-theme/pull/42/files#r301528968

File: default.hbs (hunk 1 above)
Comment (miyaliu666) [reply default.hbs]: > We would want to translate and point to the Chinese forum link here instead? But that can be taken up separate of this pull-request.

Ah yes, the same sentence appeared several times, and I translated some of them and pointed to the Chinese forum https://chinese.freecodecamp.org/forum/t/freecodecamp/374 (an introduction to Chinese News). Not sure why I didn't translate this one:(
Should I cancel this PR and submit another one?

Comment (raisedadead) [reply default.hbs]: You are able to edit the same file on the same branch, and this PR will automatically update.

**Comment by miyaliu666** [comment]
Please checked the updated.
I think we can delete social-row.hbs for now, as it encourages people to share on twitter/facebook. Later we may add: share on WeChat with scanning QR code. It may require registration on WeChat platform. I'll look into it. @raisedadead

**Review by raisedadead — APPROVED** [review]
Code LGTM. @ahmadabdolsaheb can you deploy this on Chinese news when you get a chance.

**Pull request merged**